### DISCUSSION TITLE
Fake DS4 PSM Patching

### DISCUSSION THREAD

**Discussion by Skcyte** [nefarius/BthPS3#63] [Q&A]
Just wondering if you can add these or any devices to drop its connection after the first power on? I'm no mean expert in programming but if it's adding a line of code maybe I could build it for myself.
The filter does worked but it seems it's not recognizing to drop its connection. Of course disabling PSM will make the controller connect immediately.

**Comment by nefarius** [comment]
What devices? What is a fake DS4 PSM? Sorry that entire post makes no sense whatsoever 🤔

**Comment by Skcyte** [comment]
Sorry if this confusing.
Basically the filtering used when using DS4 controller. With genuine DS4 the pairing automatically stop after you turned it on with filter enabled. But not with the fake one. I don't know if this applies to after market controller.
I'm talking about the PSM Patching in the BthPS3 Utility. Point 2 and 3.

**Reply by nefarius** [reply]
The side effects of the patch are explained in the docs site and how to mitigate them, there's nothing further to be done or possible in code.
